pyaptly/main.py

```diff
@@ -45,8 +37,25 @@ def prepare(args):
     """Set pretend mode, read config and load state."""
     command.Command.pretend_mode = args.pretend
 
+    path = Path(args.config)
     with open(args.config, "rb") as f:
-        cfg = tomli.load(f)
+        if path.suffix == ".toml":
+            import tomli
+
+            cfg = tomli.load(f)
+        elif path.suffix == ".json":
+            import json
+
+            cfg = json.load(f)
+        elif path.suffix in (".yaml", ".yml"):
+            import yaml
+
+            cfg = yaml.safe_load(f)
+            lg.warn(
+                "NOTE: yaml has beed deprecated and will be remove on the next major release."
+            )
```

Comment:
Maybe use the built in `DeprecationWarning` with https://docs.python.org/3/library/warnings.html#warnings.warn ?